blaze-client/src/test/scala/org/http4s/client/blaze/BlazePooledHttp1ClientSpec.scala

```diff
@@ -1,7 +1,31 @@
-package org.http4s.client.blaze
+package org.http4s
+package client
+package blaze
 
-import org.http4s.client.ClientRouteTestBattery
+import scalaz.concurrent.Task
 
+class BlazePooledHttp1ClientSpec
+  extends { val client = PooledHttp1Client() }
+    with ClientRouteTestBattery("Blaze PooledHttp1Client", client) {
 
-class BlazePooledHttp1ClientSpec extends ClientRouteTestBattery("Blaze PooledHttp1Client", PooledHttp1Client())
+  val path = "/simple"
```

Comment:
Exactly. I think it would make a good PR on its own. Plus this might not be the only location we see similar behavior.